umbral/utils.py

```diff
@@ -1,12 +1,16 @@
-import math
+from typing import List, Type, Union
 
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric.ec import SECP192R1, SECP256K1, SECP256R1, SECP384R1
 from cryptography.hazmat.primitives.kdf.hkdf import HKDF
-from cryptography.hazmat.primitives.asymmetric import ec
+
 from umbral import openssl
 
-def lambda_coeff(id_i, selected_ids):
+CurveTypes = Union[Type[SECP256K1], Type[SECP192R1], Type[SECP384R1], Type[SECP256R1]]
```

Comment:
We can get rid of `SECP192R1`.

Reply:
Well, it's a type that is passed in during test runtime, an it needs to be included in the annotation for it to pass type checks.  Perhaps we need curves type-aliases